webdriver/tests/support/inline.py

```diff
@@ -23,6 +24,7 @@
 }
 MIME_TYPES = {
     "html": "text/html",
+    "html_empty": "text/html",
```

Comment:
Given that this is not a MIME type lets not add it here. Instead please revert this change and provide a comment in why you had to use the quirks template. If more occurrences like this will happen in the future we may have to think about a refactoring of the inline fixture.